test/pytests/test_scripts/test_create_patches_files.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+import sys
+
+scripts_path = Path(__file__).parent.parent.parent.parent / "scripts"
+sys.path.append(str(scripts_path))
+from create_patches_files import get_patch_info
+
+
+def test_riscv_body():
+    # https://patchwork.sourceware.org/project/gcc/patch/20240611061943.698499-1-pan2.li@intel.com/
+    # patch with riscv in the body of the message
+    url = "https://patchwork.sourceware.org/api/1.3/patches/?order=date&project=6&before=2024-06-11T06:20:00&since=2024-06-11T06:15:00"
```

Comment:
In the future, we might want to turn these urls into fixture if reused

Reply:
Each url is unique and meant to pull out a single patch/patch range. I'm not super familiar with fixtures but IIUC they're mainly used to constants like api keys?

Reply:
Fixtures are basically reusable constants/objects for pytest. For now, they are all unique so it doesn't matter but just mentioning for the future Pytest files 😃